### PR TITLE
Feature/auth0

### PR DESCRIPTION
**Pull request by natesheridan**
## [DripList PR Template 🍅]

**What kind of branch was this?**
- [X] - Feature PR
- [ ] - Refactor PR
- [ ] - Other PR

**What (if any) features are you implementing?**

 - User Authentication (Auth0)
 - Lost Styles
 
**What (if anything) did you refactor?**

 - (styles)

**Were there any issues that arose?** 

 - We needed to add an Auth directory for Auth0 the video tutorial we were watching didn't use this method
 - We don't know how to access the user data

**Is there anything needs to be prioritized?**

 - Utilize user data to implement saving drinks to user data via local storage.

**Is there anything that you need from your teammates?**

 - (NA)

**GOALS?**

 - (NA)

### CODE REVIEW

**Review by annnuuuh — APPROVED** [review]

File: src/components/Rum/Rum.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Rum = () => {
     getRumRecipes()
     .then(data => data.drinks)
```
Comment (robbiejaeger) [inline src/components/Rum/Rum.js:14]: Why not combine these into one line with:
`.then(data => setRumContainer(data.drinks))`

File: src/components/Tequila/Tequila.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const Tequila = () => {
     getTequilaRecipes()
     .then(data => data.drinks)
     .then(drinks => setTequilaContainer(drinks))
```
Comment (robbiejaeger) [inline src/components/Tequila/Tequila.js:15]: Same comment as above with combining the lines.

File: src/components/Vodka/Vodka.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const Vodka = () => {
     getVodkaRecipes()
     .then(data => data.drinks)
     .then(drinks => setVodkaContainer(drinks))
```
Comment (robbiejaeger) [inline src/components/Vodka/Vodka.js:15]: Same comment as above with combining the lines.

File: src/components/Welcome/Welcome.js [hunk 4]
```diff
@@ -1,56 +1,60 @@
 import React from 'react';
 import RecipeCard from '../RecipeCard/RecipeCard.js';
 import './Welcome.css';
-import { getRandomRecipe } from './../../APICalls'
+import { getRandomRecipe, getLatestRecipes } from './../../APICalls'
+import LoginButton from '../LoginButton/LoginButton';
+
 
 const Welcome = () => {
   const  userData = {}
```
Comment (robbiejaeger) [inline src/components/Welcome/Welcome.js:9]: By looking at this line of `userData`, I'm not sure what its purpose is. Will it be used as state once the user is logged in?

File: src/components/Welcome/Welcome.js [hunk 5]
```diff
@@ -1,56 +1,60 @@
...
   
   const [randomDrink, setRandomDrink] = React.useState([])
```
Comment (robbiejaeger) [inline src/components/Welcome/Welcome.js:11]: You'll typically see `useState` or `useEffect` destructured above in the `import` using `import React, { useState, useEffect } from 'react'`. This reduces a little redundancy in saying `React.__`.

File: src/components/Welcome/Welcome.js [hunk 6]
```diff
@@ -1,56 +1,60 @@
...
+  const [drinks, setDrinks] = React.useState([])
 
   React.useEffect(() => {
     getRandomRecipe()
     .then(data => {
       setRandomDrink(data.drinks[0])
```
Comment (robbiejaeger) [inline src/components/Welcome/Welcome.js:17]: I'm not seeing any error handling path here for either of the network requests, which should be addressed if something fails server side.

File: .env [hunk 7]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=dev-ziedxe-0.us.auth0.com
```
Comment (robbiejaeger) [inline .env:1]: Nice work using environment variables here. However, you do not want to commit your `.env` file if you can help it (as to keep the keys secret from anyone looking through GitHub).

That being said, the keys will always be visible in your source code through the browser, but you can't prevent that, and that's ok.

**Pull request merged**